templates/pagination.html

```diff
@@ -1,49 +1,56 @@
 <section class="p-strip is-shallow is-bordered u-no-padding--top">
   <div class="row">
     <div class="col-12">
-      {% if current_page and total_pages and total_pages > 1 %}
+      {% if current_page and total_pages  and total_pages > 1 %}
+        {% set query_args = request.args | urlencode %}
+        {% set page_string = current_page | string %}
+        {% set query_string = request.args | urlencode | replace('&page=' + page_string, '') %}
```

Comment:
this isn't working ... started on page=7, went to page 9 and got `http://0.0.0.0:8023/search?page=7&q=lxd&page=9`